release/build/main.go

```diff
@@ -169,14 +169,22 @@ func hashreleaseSubCommands(cfg *config.Config) []*cli.Command {
 				} else if c.String(operatorRegistryFlag) != "" && c.String(operatorImageFlag) == "" {
 					return fmt.Errorf("%s must be set if %s is set", operatorImageFlag, operatorRegistryFlag)
 				}
+				if !cfg.CI.IsCI {
```

Comment:
I'm not sure I really agree with this, but it's a warning so I'm not too bothered. 

I think it's important that we can run this locally without asking the user to make any changes, so that we can actually test the behavior that gets run in CI.